mlrun/model_monitoring/api.py

```diff
@@ -568,9 +576,14 @@ def get_sample_set_statistics(
         # Return the statistics logged with the model:
         return model_artifact_feature_stats
 
-    # Turn the DataItem to DataFrame:
-    if isinstance(sample_set, mlrun.DataItem):
-        sample_set, _ = read_dataset_as_dataframe(dataset=sample_set)
+    # Turn other object types to DataFrame:
+    if isinstance(sample_set, (mlrun.DataItem, list, tuple, dict, np.ndarray)):
```

Comment:
This list doesn't quite match the [list of types supported by read_dataset_as_dataframe()](https://github.com/mlrun/mlrun/blob/v1.6.0-rc21/mlrun/model_monitoring/api.py#L590).

Reply:
Change docs.
About URI - I will check with @Eyal-Danieli .